OpenDreamShared/Compiler/DMPreprocessor/DMPreprocessorLexer.cs

```diff
@@ -194,6 +194,11 @@ class DMPreprocessorLexer : TextLexer {
                 if (_source[_currentPosition] == '\n') { //Skip a newline if it comes after a backslash
                     base.Advance();
 
+                    while (current == ' ' || current == '\t' || current == '\n')
+                    {
+                        current = Advance();
+                    }
+                    
                     current = Advance();
```

Comment:
This line should be removed, it ends up skipping the first non-whitespace character